src/components/ElasticQueryField/ElasticQueryField.js

```diff
@@ -56,20 +75,37 @@ const ElasticQueryField = props => {
   const [options, setOptions] = useState([]);
   const [focusedOptionIndex, setFocusedOptionIndex] = useState(UNSELECTED_OPTION_INDEX);
   const [typedValue, setTypedValue] = useState('');
+  const [typedValueForEditMode, setTypedValue2] = useState('');
```

Comment:
Please change the naming for the setter to be consistent 
const [typedValueForEditMode, setTypedValueForEditMode] = useState('');

Reply:
Thanks, fixed